test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -504,6 +507,42 @@ name: lua
   cleanup();
 }
 
+// Test whether respond api will call response_headers_to_add.
```

Comment:
`Test whether the 'response_headers_to_add' is valid for the Lua 'respond' method.` ?